src/platform/user/package.json

```diff
@@ -29,7 +29,7 @@
     "./SessionTimeoutModal": "./authentication/components/SessionTimeoutModal.jsx",
     "./SignInModal": "./authentication/components/SignInModal.jsx",
     "./VerifyAccountLink": "./authentication/components/VerifyAccountLink.jsx",
-    "./VerifyAlert": "./authentication/components/VerifyAlert.jsx",
+    "./VerifyAlert": "./authorization/components/VerifyAlert.jsx",
```

Comment:
This is a valid change, there is no `src/platform/user/authentication/components/VerifyAlert.jsx`

Reply:
Making this change so that the cypress tests will actually run